### DISCUSSION TITLE
[Unstable] "psychosisartifact_noeffect" variant affliction error.

### DISCUSSION THREAD

**Discussion by Ek-Videogames** [FakeFishGames/Barotrauma#14684] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

![image](https://github.com/user-attachments/assets/f5a9cdcf-63d2-4e1e-b587-4df87ea0ca70)

The game doesn't seem to like that the new overwritten variant of the holdable status effect doesn't have any affliction identifier.

### Reproduction steps

A structure spawns somewhere in the level which uses this 'decorative' psychosis artifact.

### Bug prevalence

Happens every now and then

### Single player or multiplayer?

Single player

### Version

v1.6.4.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

### Relevant error messages and crash reports

```
  <Item name="" identifier="psychosisartifact_noeffect" nameidentifier="psychosisartifact" descriptionidentifier="decorativepiece" variantof="psychosisartifact" hideinmenus="true">
    <Holdable>
      <StatusEffect />
      <StatusEffect>
        <Affliction strength="0"/>
        <Affliction strength="0" />
      </StatusEffect>
    </Holdable>
  </Item>
```

**Comment by Regalis11** [comment]
This has been addressed since the report was created, looks like we just forgot to close the ticket. :)
